### ISSUE TITLE
Last three menu items missing

### ISSUE THREAD

**Issue by Digitalclips** [divamgupta/diffusionbee-stable-diffusion-ui#471]
On my Mac Studio M2 Ultra running Sonoma developer release 14, I do not see the Lora or anything below that on the left menu selections in v2.2.1.

**Comment by slam-campup** [comment]
Same here. Missing everything below Settings, including LORA tools, interpolator, and training.

**Comment by philmodin** [comment]
Go to Home tab, then scroll down to Miscellaneous.

**Comment by Digitalclips** [comment]
First of all, Miscellaneous isn't present in the Home menu (not a tab in v 2.2.1), only in Text to Image and Image to Image menu selections.  However, Miscellaneous doesn't offer any way to get the Lora and lower menu options to show up for my v2.2.1 installation.  Are you using v2.2.1 when offering this suggestion?

**Comment by slam-campup** [comment]
I found it in v2.2.1.

1. Go to the Home screen, it may default to this page or you may need to go to Tools > Home
2. Scroll down to the Miscellaneous row at the bottom of the screen
3. Find the boxes for LoRA Tools, Interpolator, Deforum Video
4. Press the Open button on one of these tools 
5. The matching menu will now appear in the Tools sidebar

**Comment by Digitalclips** [comment]
YES!!!  Thank you.  The problem for me was twofold: there was no need to scroll on a 27" 4K screen; the three items were in front of my eyes all the time!  However, the toggles show up only in Dark Mode, and I don't use Dark Mode.
